psydac/feec/tests/test_global_projectors.py

```diff
@@ -52,16 +55,22 @@ def test_H1_projector_1d(domain, ncells, degree, periodic):
 @pytest.mark.parametrize('domain', [(0, 2*np.pi)])
 @pytest.mark.parametrize('ncells', [100, 200, 300])
 @pytest.mark.parametrize('degree', [2])
-@pytest.mark.parametrize('periodic', [False])
+@pytest.mark.parametrize('periodic', [True,False])
 @pytest.mark.parametrize('nquads', [100, 120, 140, 160])
+@pytest.mark.parametrize('multiplicity', [1,2])
 
-def test_L2_projector_1d(domain, ncells, degree, periodic, nquads):
+def test_L2_projector_1d(domain, ncells, degree, periodic, nquads, multiplicity):
 
+    #change mulitplicity if higher than degree to avoid problems (case p<m doesn't work)
+    multiplicity = min(multiplicity,degree)
     breaks = np.linspace(*domain, num=ncells+1)
-    knots  = make_knots(breaks, degree, periodic)
+    knots  = make_knots(breaks, degree, periodic, multiplicity=multiplicity)
 
     domain_decomposition = DomainDecomposition([ncells], [periodic])
 
+    # H1 space (0-forms)
+    #change mulitplicity if higher than degree to avoid problems (case p<m doesn't work)
+    multiplicity = min(multiplicity,degree)
```

Comment:
Typo: mulitplicity -> multiplicity

More readable:
```suggestion
    #change multiplicity if higher than degree to avoid problems (case p<m doesn't work)
    multiplicity = min(multiplicity, degree)
```